optimum/exporters/onnx/config.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Common onnx configuration classes that handle most of features for building model specific configurations."""
+
+from collections import OrderedDict
+from typing import Mapping
+
+from ...utils import (
+    DummyDecoderTextInputGenerator,
+    DummyPastKeyValuesGenerator,
+    DummySeq2SeqPastKeyValuesGenerator,
+    DummyTextInputGenerator,
+)
+from .base import OnnxConfig, OnnxConfigWithPast, OnnxSeq2SeqConfigWithPast
+
+
+class AutoEncoderOnnxConfig(OnnxConfig):
```

Comment:
`EncoderOnnxConfig` is a better name no? To me an auto-encoder is a encoder-decoder type of architecture (or at least that's how it is understood in computer vision :grin:).

Reply:
Changed, thanks for the remark!